packages/server/graphql/types/Organization.ts

```diff
@@ -141,10 +142,9 @@ const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<a
       type: new GraphQLNonNull(OrganizationUserConnection),
       resolve: async ({id: orgId}, _args, {dataLoader}) => {
         const organizationUsers = await dataLoader.get('organizationUsersByOrgId').load(orgId)
-        organizationUsers.sort((
-          a: OrganizationUser,
-          b: OrganizationUser
-        ) => (a.orgId > b.orgId ? 1 : -1))
+        organizationUsers.sort((a: OrganizationUser, b: OrganizationUser) =>
```

Comment:
Ahh sorry, it looks like there's a bunch of linting changes after merging with master. My prettier settings haven't changed, and I don't see any eslint/prettier node module upgrades. Perhaps I hit save all while merging, and that triggered some linting changes 🤷‍♂️

Reply:
Ahh yes, it looks like `"printWidth": 100` wasn't applied in [that PR](https://github.com/ParabolInc/parabol/pull/5403/files). @tiffanyhan do you know whether you have a vscode plugin that might have changed the `printWidth` in that PR?

Reply:
ah my fault, i was on a fresh install of vscode and didn't have prettier installed. installed now, adding linting to github actions sounds nice too